setup.py

```diff
@@ -27,7 +27,7 @@ def read_file(filename):
     'python-dateutil',
     'pyramid_multiauth >= 0.8',  # User on policy selected event.
     'transaction < 2',  # 2.X is incompatible with pyramid_tm
-    'pyramid_tm',
+    'pyramid_tm < 1.1.0',
```

Comment:
We should probably wait for 1.1.1 which will be there shortly now. https://github.com/Pylons/pyramid_tm/pull/52